src/libsyntax_pos/lib.rs

```diff
@@ -798,6 +818,27 @@ impl FileMap {
         self.multibyte_chars.borrow_mut().push(mbc);
     }
 
+    pub fn record_width(&self, pos: BytePos, ch: char) {
+        let width = match ch {
+            '\t' | '\n' =>
+                // Tabs will be printed as half-width spaces.
```

Comment:
Is U+0020 really a full-width space? I'm not sure about the terminology, but its not a specially narrow character or something.

Reply:
Oh sorry disregard, its a specific Unicode term. The comment is okay.